torchci/lib/bot/autoLabelBot.ts

```diff
@@ -1,10 +1,71 @@
 import { Context, Probot } from "probot";
 
-const regexToLabel: [RegExp, string][] = [
+const titleRegexToLabel: [RegExp, string][] = [
   [/rocm/gi, "module: rocm"],
   [/DISABLED\s+test.*\(.*\)/g, "skipped"],
 ];
 
+const filenameRegexToReleaseCategory: [RegExp, string][] = [
+  // dataloader_frontend => does that correlate best with release notes: AO frontend?
+  [/torch\/utils\/data/gi, "module: dataloader"],
+  [/test_data(loader|pipe)/gi, "module: dataloader"],
```

Comment:
is module: dataloader correct?

Reply:
See comment below, we probably want to use the "release notes:" prefix